content/docs/sidebar.json

```diff
@@ -94,13 +94,18 @@
     "slug": "user-guide",
     "source": "user-guide/index.md",
     "children": [
+      {
+        "label": "Glossary",
+        "slug": "glossary",
+        "source": "basic-concepts"
```

Comment:
I wasn't sure where to have the "Edit on Github" link take you since there isn't a single page. I linked the page to the [`basic-concepts`](https://github.com/iterative/dvc.org/tree/master/content/docs/user-guide/basic-concepts) folder for now.

Reply:
makes most sense to me, yes.